api/db/user.js

```diff
@@ -70,7 +78,6 @@ module.exports = (zxcvbn = defaultZxcvbn, bcrypt = defaultBcrypt) => ({
         }
 
         logger.silly('password is sufficiently complex; hashing it');
-        this.set({ password: bcrypt.hashSync(this.attributes.password) });
```

Comment:
This was hashing the password in the `validate()` function, which seems to make sense, except `save()` also calls `validate()` - so if you validate the model independent of saving it, you end up double-hashing the password.  And then you can never log in again.  So...  That's no good.